### PR TITLE
SONARGO-58 Clean up integration tests to only include Go

### PR DESCRIPTION
**Pull request by petertrr**
[SONARGO-58](https://sonarsource.atlassian.net/browse/SONARGO-58)

Current master: `4349 source files to be analyzed`
With this change: `3288 source files to be analyzed`

[SONARGO-58]: https://sonarsource.atlassian.net/browse/SONARGO-58?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: its/ruling/src/test/resources/sources/build.gradle [hunk 1]
```diff
@@ -3,6 +3,6 @@ plugins {
 }
 
 golang {
```
Comment (mstachniuk) [inline its/ruling/src/test/resources/sources/build.gradle]: Is this file needed at all? The `gradle tasks` shows me the following error:
```
Project directory '.../sonar-go/its/ruling/src/test/resources/sources/go' is not part of the build defined by settings file '.../sonar-go/settings.gradle'. If this is an unrelated build, it must have its own settings file.
```

Maybe it should be removed?

**Review by mstachniuk — APPROVED** [review]
LGTM

File: its/ruling/src/test/resources/sources/build.gradle (hunk 1 above)
Comment (petertrr) [reply its/ruling/src/test/resources/sources/build.gradle]: Yes, I can see that it exists from the very beginning of the Go analyzer without a clear usage. And the `gogradle` plugin is archived for two years already. I'll remove the file
